_posts/2020-10-04-newpipe-0.20.0.md

```diff
@@ -1,154 +1,176 @@
+Talking about sharing, when you share a live stream with someone, NewPipe doesn't add a timestamp to the URL any more. This was done by [@nmurali94](https://github.com/nmurali94). Thanks :)
 
-The header layout has been improved: the alignment on the sides is consistent now and longer headings can fit themselves on two lines instead of just one.
+The header layout has improved: The alignment on the sides is consistent, and longer headings can expand to two lines.
 
-If you ever tried to search something and then lost connection to the internet, or just tried to search while being offline, you would have noticed that NewPipe didn't let you type your search term. Instead it kept on rudely interrupting you with a network error. That is fixed now. You can type all you want, and it will only fail if you actually hit enter or press the search button. Both of these changes were brought to you by [@Stypox](https://github.com/Stypox). Thanks!
+The thing worse than searching while offline, or losing your connection to the Internet when doing it, is that NewPipe didn't let you type your search term.
+Being rudely interrupted by a network error is a thing of the past. Type all you want, and accept defeat after the fact. Both of these changes brought to you by [@Stypox](https://github.com/Stypox). Thanks!
 
-In order to save some of that valuable mobile data, you were already able to turn off video thumbnails and profile pictures for comments. Now even the black placeholders which replaced them are gone and comments can comfortably take up the space left behind. Thanks, [@4D17Y4](https://github.com/4D17Y4)!
+To save valuable mobile data, turning off video thumbnails and profile pictures for comments was already possible. Now even the black placeholders which replaced them are gone, and comments can comfortably take up the space left over. Thanks, [@4D17Y4](https://github.com/4D17Y4)!
 
-[@TobiGr](https://github.com/TobiGr) brought two more improvements into this release: the setting to remember pop-up size and position has a less redundant description, and when you switch to another app, the main player now minimizes to background audio by default to allow for a more seamless transition. Swoosh.
+[@TobiGr](https://github.com/TobiGr) brought two more improvements: The setting to remember pop-up size and position has a less redundant description, and when switching to another app, the main player now minimizes to play audio in the background by default. Seamless transition. Swoosh.
 
-It's 2020, and our default resolutions also realized this. Instead of having 360p everywhere, main player videos are opened in 720p at 60FPS now, and pop-up ones in 480p. Thank you, [@B0pol](https://github.com/B0pol). NewPipe now also remembers what kind of media you downloaded the last time, and will auto-select it when you hit download on a new video. This was done by [@vmazoyer](https://github.com/vmazoyer).
```

Comment:
2020 default resolutions, attempt but no cigar. unreferenced "Download".

Reply:
> attempt but no cigar.

Waaaaaat?

> unreferenced "Download".

Should have been the next paragraph. This @Poolitzer I tell you. Doesn't listen to me at all. Damn you, Poolitzer! xD

Reply:
let me put two things in one paragraph got damn it.

Reply:
Nyo. ಠ╭╮ಠ